spec/design/skel/AARCH64/ArchStructures_H.thy

```diff
@@ -54,10 +54,8 @@ where
 
 end
 
-context begin interpretation Arch .
-
-requalify_types
+(* not possible to move this requalification to generic, as some arches don't have vcpu *)
```

Comment:
I'm not sure if I actually like this idea, but would it be possible to have a 'semi-generic' file that is only imported by an `ARM_HYP` and `AARCH64` file and so isn't visible on other architectures?

Reply:
This idea has merit, especially if we ever want to go for the golden chalice that is being generic in vcpu on/off, but I think right now that's not going to gain us much for these small blips.

Reply:
Agreed, not worth doing now, just a thought I wanted to get out of my head,

Reply:
Agreed, should leave that for later, for if/when we try for generic VCPU on/off.